vllm/engine/arg_utils.py

```diff
@@ -911,13 +911,21 @@ def create_model_config(self) -> ModelConfig:
             override_generation_config=self.override_generation_config,
             enable_sleep_mode=self.enable_sleep_mode,
             model_impl=self.model_impl,
+            using_tensorizer=self.load_format == LoadFormat.TENSORIZER,
         )
 
     def create_load_config(self) -> LoadConfig:
 
         if self.quantization == "bitsandbytes":
             self.load_format = "bitsandbytes"
 
+        if self.load_format == "tensorizer" and not self.model_loader_extra_config:
```

Comment:
I feel like this should infer unless the `self.model_loader_extra_config.tensorizer_dir` or `tensorizer_uri` has been set.  Right now setting any config option stops this.